ownCloud/Settings/MediaUploadSettingsSection.swift

```diff
@@ -72,6 +155,180 @@ class MediaUploadSettingsSection: SettingsSection {
 
 		self.add(row: convertPhotosSwitchRow!)
 		self.add(row: convertVideosSwitchRow!)
+
+		// Instant upload requires at least one configured account
+		OCBookmarkManager.shared.loadBookmarks()
```

Comment:
Not sure if this `loadBookmarks()` call is needed. They should already have been loaded at that point.

Reply:
@felix-schwarz I think, that doesn't happen necessarily if the first thing user does after opening the app is navigating to settings menu